### DISCUSSION TITLE
How to get images/digests for images built with ko ?

### DISCUSSION THREAD

**Discussion by eddycharly** [goreleaser/goreleaser#4029] [Q&A] [answered]
We use ko to build docker images, however we need the images and digests to generate SLSA attestations.

Is this possible to get this info after goreleaser has run ?

**Comment by developer-guy** [comment]
This release workflow could help: _https://github.com/sigstore/cosign-gatekeeper-provider/pull/32/files_

**Reply by eddycharly** [reply]
This workflow doesn't seem to generate provenance for built images (only for binaries) right ?

**Comment by developer-guy** [comment]
Or do you have open issue for that on Kyverno side, you can send it over here, I'd like to work on it.

**Reply by eddycharly** [reply]
We don't have an issue yet.
This is for the kyverno playground project (https://github.com/kyverno/playground).

We already have provenance for the built binaries, now we are trying to extend it to built images (https://github.com/kyverno/playground/blob/main/.github/workflows/release-goreleaser.yaml)

**Comment by caarlos0** [comment, marked as answer]
you can parse `dist/digest.json`, and the images should have the digest there.
